website/pages/en/cookbook/upgrading-a-subgraph.mdx

```diff
@@ -10,98 +10,39 @@ The process of upgrading is quick and your subgraphs will forever benefit from t
 
 ### Prerequisites
 
-- You have already deployed a subgraph on the hosted service.
-- The subgraph is indexing a chain available on The Graph Network.
-- You have a wallet with ETH to publish your subgraph on-chain.
-- You have ~3,000 GRT to curate your subgraph so Indexers can begin indexing it.
+- You have a subgraph deployed on the hosted service.
 
 ## Upgrading an Existing Subgraph to The Graph Network
 
-> You can find specific commands for your subgraph in the [Subgraph Studio](https://thegraph.com/studio/).
+<VideoEmbed youtube="MLqJdVCom5k" />
 
-1. Get the latest version of the graph-cli installed:
+If you are logged in to the hosted service, you can access a simple flow to upgrade your subgraphs from [your dashboard](https://thegraph.com/hosted-service/dashboard), or from an individual subgraph page.
 
-```sh
-npm install -g @graphprotocol/graph-cli
-```
-
-```sh
-yarn global add @graphprotocol/graph-cli
-```
-
-Make sure your `apiVersion` in subgraph.yaml is `0.0.5` or greater.
-
-2. Inside the subgraph's main project repository, authenticate the subgraph to deploy and build on the studio:
-
-```sh
-graph auth --studio <DEPLOY_KEY>
-```
+> This process typically takes less than five minutes.
 
-3. Generate files and build the subgraph:
+1. Select the subgraph(s) you want to upgrade.
+2. Connect or enter the receiving wallet (the wallet that will become the owner of the subgraph).
+3. Click the "Upgrade" button.
 
-```sh
-graph codegen && graph build
-```
+That's it! Your subgraphs will be deployed to the Subgraph Studio, and published on The Graph Network. You can access the [Subgraph Studio](thegraph.com/studio) to manage your subgraphs, logging in with the wallet specified during the upgrade process.
```

Comment:
Nitpick, but is it really "the" Subgraph Studio or just Subgraph Studio? cc @bermchain